### PR TITLE
Hotfix: #3 always allow magic non interface methods

### PR DESCRIPTION
**Pull request by Ocramius**
Provides fixes for #3

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "InterNations\\Component\\TypeJail\\Tests\\Fixtures\\": "tests/InterNations/Fixtures"
+            "InterNations\\Component\\TypeJail\\Tests\\": "tests/InterNations"
```
Comment (Ocramius) [inline composer.json:26]: I corrected this, or else I wouldn't be able to run single tests

File: phpunit.xml.dist [hunk 2]
```diff
@@ -25,6 +25,9 @@
     </logging>
 
     <filter>
+        <whitelist addUncoveredFilesFromWhitelist="true">
```
Comment (Ocramius) [inline phpunit.xml.dist:28]: Also corrected: phpunit was returning a warning otherwise

**Comment by lstrojny** [comment]
Can you fix the coding style, then we are ready to merge

**Comment by Ocramius** [comment]
@lstrojny pushing. Let's patiently wait for the build :-)

**Comment by Ocramius** [comment]
@lstrojny green!

**Comment by lstrojny** [comment]
Thank you!

**Comment by Ocramius** [comment]
\o/

**Pull request merged**